cypress/integration/article_spec.js

```diff
@@ -0,0 +1,21 @@
+describe('News Article', () => {
+const url = 'http://localhost:3000/';
+
+    it('should display 200/OK', () => {
+        cy.request(url + 'status').then((response) => {
```

Comment:
Can this be `await` rather than a promise chain?

Reply:
Maybe I'm reading this wrong I don't think we can use `await` - "Without retry-ability, assertions would randomly fail. This would lead to flaky, inconsistent results. This is also why we cannot use new JS features like async / await." from https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#How-Cypress-really-looks-Promises-wrapped-up-and-hidden-from-us